config/v1/types_cluster_version.go

```diff
@@ -225,7 +225,7 @@ type UpdateHistory struct {
 type ClusterID string
 
 // ClusterVersionCapability enumerates optional, core cluster components.
-// +kubebuilder:validation:Enum=openshift-samples
+// +kubebuilder:validation:Enum=openshift-samples;cluster-baremetal-operator
```

Comment:
After [this thread][1], you moved the constant value to `baremetal`, but you need to update this enum declaration to match.

[1]: https://github.com/openshift/api/pull/1125#discussion_r818158011